packages/binding-coap/src/coap-server.ts

```diff
@@ -228,21 +231,45 @@ export default class CoapServer implements ProtocolServer {
         return opValues;
     }
 
+    private addFormToAffordance(form: TD.Form, affordance: AffordanceElement): void {
+        const affordanceForms = affordance.forms;
+        if (affordanceForms == null) {
+            affordance.forms ??= [form];
```

Comment:
Am I lost or this null coalescing operator is redundant?

Reply:
True, the `??=` is not actually needed here :) Simplified it in 7109dcb913b4d3a8cb61caa5662751358767128d